src/tests/ftest/server/metadata_svc_ops.yaml

```diff
@@ -0,0 +1,56 @@
+hosts:
+  test_servers: 3
+  test_clients: 1
+timeout: 850
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      targets: 8
+      nr_xs_helpers: 4
+      first_core: 0
+      pinned_numa_node: 0
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
+        - RDB_COMPACT_THRESHOLD=64
+        - DD_MASK=group_metadata_only
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem0"]
+          scm_mount: /mnt/daos0
+    1:
+      targets: 8
+      nr_xs_helpers: 4
+      first_core: 0
+      pinned_numa_node: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      log_mask: DEBUG,MEM=ERR
```

Comment:
want to have log_mask: ERR when running a performance-sensitive test

Reply:
Please see the new commit.